### PR TITLE
Rename the close button at the bottom of the modal window

### LINKED ISSUE #2646: modalIFrame: OK button not displayed
**Affected version(s)**

Contao 4.9

**Description**
1) File manager
2) open info box
3) modalIFrame does not display the OK-Button (see 4)
grafik
(screen shot from demo.contao.org)

Does it have any reason? Or is it a bug? Is it possible to display this button via parameter like title, width and height?

```
onclick="Backend.openModalIframe({'title':'content-images','url':this.href});return false"
```

### PR DESCRIPTION
**Pull request by leofeyer**
| Q                | A
| -----------------| ---
| Fixed issues     | Fixes #2646
| Docs PR or issue | -

@contao/developers Are we sure that the double close button provides a better UX?

### CODE REVIEW

**Comment by aschempp** [comment]
To me, only modals with an action should have buttons. The info modal does not have an action, it should be closed by clicking the cross or the background overlay (like anywhere else on _the internet®_ ).

**Comment by leofeyer** [comment]
As discussed in Mumble on June 2nd, we do not want to add a close button and instead rename the existing "Close" button to "Cancel".

**Comment by leofeyer** [comment]
Changed in b6904ab69b404c9de0e2ca9c920ea715dab8060c.

File: core-bundle/src/Resources/contao/classes/BackendTemplate.php [hunk 1]
```diff
@@ -180,7 +180,7 @@ protected function getLocaleString()
 			'var Contao={'
 				. 'theme:"' . Backend::getTheme() . '",'
 				. 'lang:{'
-					. 'close:"' . $GLOBALS['TL_LANG']['MSC']['close'] . '",'
```
Comment (Toflar) [inline core-bundle/src/Resources/contao/classes/BackendTemplate.php:183]: I would keep both here. I'm sure someone else is using the close label.

**Review by Toflar — CHANGES_REQUESTED** [review]

File: core-bundle/src/Resources/contao/classes/BackendTemplate.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Resources/contao/classes/BackendTemplate.php:183]: I am not a big fan of sending data that is nowhere used, but for the sake of BC, I have re-added the label in 9296b6640ade7e23768e6f9008ff67123dfeb42d.

**Review by Toflar — APPROVED** [review]

**Pull request merged**